backend/src/BuiltinExecution/Libs/Int8.fs

```diff
@@ -0,0 +1,323 @@
+module BuiltinExecution.Libs.Int8
+
+open FSharp.Control.Tasks
+open System.Threading.Tasks
+
+open System.Numerics
+
+open Prelude
+open LibExecution.RuntimeTypes
+open LibExecution.Builtin.Shortcuts
+
+module VT = ValueType
+module Dval = LibExecution.Dval
+
+let types : List<BuiltInType> = []
+let constants : List<BuiltInConstant> = []
+
+module ParseError =
+  type ParseError =
+    | BadFormat
+    | OutOfRange
+
+  let toDT (e : ParseError) : Dval =
+    let (caseName, fields) =
+      match e with
+      | BadFormat -> "BadFormat", []
+      | OutOfRange -> "OutOfRange", []
+
+    let typeName = TypeName.fqPackage "Darklang" [ "Stdlib"; "Int8" ] "ParseError" 0
+    DEnum(typeName, typeName, [], caseName, fields)
+
+
+let fn = fn [ "Int8" ]
+
+let fns : List<BuiltInFn> =
+  [ { name = fn "add" 0
+      typeParams = []
+      parameters = [ Param.make "a" TInt8 ""; Param.make "b" TInt8 "" ]
+      returnType = TInt8
+      description = "Adds two intergers8 together"
+      fn =
+        (function
+        | state, _, [ DInt8 a; DInt8 b ] ->
+          let result = int a + int b
+          if result < -128 || result > 127 then
+            Int.IntRuntimeError.Error.OutOfRange
+            |> Int.IntRuntimeError.RTE.toRuntimeError
+            |> raiseRTE state.caller
+            |> Ply
+          else
+            Ply(DInt8(int8 result))
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplemented
+      previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "subtract" 0
+      typeParams = []
+      parameters = [ Param.make "a" TInt8 ""; Param.make "b" TInt8 "" ]
+      returnType = TInt8
+      description = "Subtracts two integers8"
+      fn =
+        (function
+        | state, _, [ DInt8 a; DInt8 b ] ->
+          let result = int a - int b
+          if result < -128 || result > 127 then
+            Int.IntRuntimeError.Error.OutOfRange
```

Comment:
Here, I'm using the Int64 OutofRange RTE. Should we consider creating a separate RTE for Int8?